packages/fabric8-ui/src/app/dashboard-widgets/space-settings-widget/space-settings-widget.component.less

```diff
@@ -0,0 +1,61 @@
+@import (reference) '../../../assets/stylesheets/shared/osio.less';
+
+.f8-small-card {
+  height: 200px;
```

Comment:
I don't think we need to set the height for the ```space-settings-widget``` card. Because we have content of the card is not dynamic.

Reply:
removed it